### PR TITLE
feat: Provide an annotation and processor that generates application …

### PR DESCRIPTION
**Pull request by iocanel**
…crd custom resources.

### CODE REVIEW

File: annotations/application-crd-annotations/src/main/java/io/ap4k/application/annotation/Info.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+**/
+
+package io.ap4k.application.annotation;
+
+public @interface Info {
```
Comment (cmoulliard) [inline annotations/application-crd-annotations/src/main/java/io/ap4k/application/annotation/Info.java]: FYI. I have opened a ticket on k8s Application CRD project in order to have the possibility to pass k8s objects here such as List of Env, .... to let by example the operator to configure the Deployment/Deploymentconfig. The existing field `valueFrom` will select either a secret, configMap, ingress, ... resource

**Comment by cmoulliard** [comment]
Where do you define the `ComponentGroupKinds` ? @iocanel

**Comment by iocanel** [comment]
You don't! They are detected based on what has been generated.

**Comment by cmoulliard** [comment]
Great

**Pull request merged**